src/store/actions.ts

```diff
@@ -24,4 +24,13 @@ export default {
       context.dispatch(ActionType.ERROR_OCCURED, { message, details: error })
     }
   },
+
+  async LOAD_INGREDIENTS(context: Context) {
+    const response = await fetch("/api/ingredients")
+    const data = response.headers.get("content-type")?.match(/json/) ? await response.json() : await response.text()
+    if (!response.ok) {
+      throw { message: response.status + " " + response.statusText, details: data }
+    }
+    context.commit(MutationType.LOADED_INGREDIENTS, { ingredients: data.ingredients })
+  },
 }
```

Comment:
After merging #6, this should be refactored to use the new `catchErrors()` function and a separate api module for ingredients.

Reply:
done.